src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,16 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userservice = new UserService();
```

Comment:
we should follow the lower camelCase naming convention for variables
```suggestion
    private UserService userService = new UserService();
```